src/scripts/new-jenkins-notifier.js

```diff
@@ -147,8 +147,8 @@ module.exports = function(robot) {
                         makeSound('mechaloses');
                         robot.messageRoom('qa-chat', "Mecha Godzilla has been defeated!:excited_tomato::excited_tomato::excited_tomato:");
                         robot.messageRoom('qa-chat', "http://media.giphy.com/media/33T28J1KEkRq0/giphy.gif");
-                        robot.messageRoom('qa-chat', "http://media.giphy.com/media/yygLBxnwBVynu/giphy.gif");
-                        robot.messageRoom('qa-chat', "http://media.giphy.com/media/tEgJl9nSsfTGM/giphy.gif");
+                        robot.messageRoom('qa-chat', "http://i.imgur.com/XhKzx1I.gif");
+                        robot.messageRoom('qa-chat', "http://i.imgur.com/CQu0ArZ.gif);
```

Comment:
I am using the website - the "Blue" and "Black" aren't contrasting enough I
guess.

On Wed, Apr 15, 2015 at 11:56 AM, Shad Downey <notifications@github.com>
wrote:

> In src/scripts/new-jenkins-notifier.js
> <https://github.com/1stdibs/hubot-scripts/pull/6#discussion_r28433665>:
>
> > @@ -147,8 +147,8 @@ module.exports = function(robot) {
> >                          makeSound('mechaloses');
> >                          robot.messageRoom('qa-chat', "Mecha Godzilla has been defeated!:excited_tomato::excited_tomato::excited_tomato:");
> >                          robot.messageRoom('qa-chat', "http://media.giphy.com/media/33T28J1KEkRq0/giphy.gif");
> > -                        robot.messageRoom('qa-chat', "http://media.giphy.com/media/yygLBxnwBVynu/giphy.gif");
> > -                        robot.messageRoom('qa-chat', "http://media.giphy.com/media/tEgJl9nSsfTGM/giphy.gif");
> > +                        robot.messageRoom('qa-chat', "http://i.imgur.com/XhKzx1I.gif");
> > +                        robot.messageRoom('qa-chat', "http://i.imgur.com/CQu0ArZ.gif);
>
> typo here (missing closing double quote)
> what IDE are you using?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/1stdibs/hubot-scripts/pull/6/files#r28433665>.
>



-- 
-- 
*Jason Smiley*
Senior QA Automated Tester
*1stdibs*
*m  *908.433.7747
e  jason.smiley@1stdibs.com

*1stdibs <http://www.1stdibs.com/> -* *T**he Most Beautiful Things On
Earth.*
Follow us on Facebook <https://www.facebook.com/1stdibs>, Pinterest
<http://pinterest.com/1stdibs/> and Twitter <https://twitter.com/1stdibs>.